### PR TITLE
add missing argument for html message

### PR DESCRIPTION
**Pull request by DxTa**
**Current issue:**
html message is not rendered properly (it's in plain at the moment)

**Solution:**
After checking `site/messages.html` from here https://github.com/websauna/websauna/blob/master/websauna/system/core/templates/site/messages.html . I think `rich=True` should be added in case of html messages

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
         'test': [
             'codecov',
             'flake8',
-            'pytest>=3.0',
+            'pytest>=3.0,<4.0',
```
Comment (DxTa) [inline setup.py:67]: related to this issue from `pytest-splinter` https://github.com/pytest-dev/pytest-splinter/issues/112

**Comment by ericof** [comment]
@DxTa Could you please fix the lint issue?

**Comment by DxTa** [comment]
> @DxTa Could you please fix the lint issue?

@ericof done. Let me know if I might miss anything

**Review by ericof — APPROVED** [review]
LGTM

**Pull request merged**